.gitlab-ci.ornl.yml

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
general comment for this CI config file - move it into `/.gitlab` directory

Reply:
Done